src/Data/Handler.php

```diff
@@ -54,12 +45,8 @@ public static function read(string $file): array
 
 	/**
 	 * Writes data to a file
-	 *
-	 * @param string $file
-	 * @param mixed $data
-	 * @return bool
 	 */
-	public static function write(string $file = null, $data = []): bool
+	public static function write(string|null $file = null, mixed $data = []): bool
```

Comment:
Here we can even change it without a breaking change: `F::write()` only accepts `string`.

```suggestion
	public static function write(string $file, mixed $data = []): bool
```